### PR TITLE
Add country name resolving by code.

### PR DESCRIPTION
**Pull request by gmaf**

### CODE REVIEW

File: src/Lykke.Common/Abstractions/ICountryNameResolver.cs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+﻿namespace Lykke.Common.Abstractions
+{
+    public interface ICountryNameResolver
```
Comment (starkmsu) [inline src/Lykke.Common/Abstractions/ICountryNameResolver.cs]: Add some XML comments to public type

File: src/Lykke.Common/CountryManager.cs [hunk 2]
```diff
@@ -336,5 +593,15 @@ public static string Iso2ToIso3(string iso2)
             return CountryIso2ToIso3Links.ContainsKey(iso2) ? CountryIso2ToIso3Links[iso2] : iso2;
         }
 
+        public static string GetCountryNameByIso3(string iso3)
```
Comment (starkmsu) [inline src/Lykke.Common/CountryManager.cs]: Add some XML comments to public method

File: src/Lykke.Common/CountryManager.cs [hunk 3]
```diff
@@ -336,5 +593,15 @@ public static string Iso2ToIso3(string iso2)
...
+        {
+            return CountryIso3ToNameLinks.ContainsKey(iso3) ? CountryIso3ToNameLinks[iso3] : string.Empty;
+        }
+
+        public static string GetCountryNameByIso2(string iso2)
```
Comment (starkmsu) [inline src/Lykke.Common/CountryManager.cs]: Same as above

File: src/Lykke.Common/CountryManager.cs [hunk 4]
```diff
@@ -300,6 +300,263 @@ public static class CountryManager
 
         };
 
+        public static readonly Dictionary<string, string> CountryIso3ToNameLinks = new Dictionary<string, string>
```
Comment (KonstantinRyazantsev) [inline src/Lykke.Common/CountryManager.cs]: Make it IReadOnlyDictionary and all other properties in this class too

File: src/Lykke.Common/CountryManager.cs [hunk 5]
```diff
@@ -336,5 +593,15 @@ public static string Iso2ToIso3(string iso2)
             return CountryIso2ToIso3Links.ContainsKey(iso2) ? CountryIso2ToIso3Links[iso2] : iso2;
         }
 
+        public static string GetCountryNameByIso3(string iso3)
+        {
+            return CountryIso3ToNameLinks.ContainsKey(iso3) ? CountryIso3ToNameLinks[iso3] : string.Empty;
```
Comment (KonstantinRyazantsev) [inline src/Lykke.Common/CountryManager.cs]: could be ```CountryIso3ToNameLinks.TryGetValue(iso3, var out name) ? name : string.Empty;```

**Pull request merged**